docs/howto/edit-in-protege.md

```diff
@@ -0,0 +1,74 @@
+# Editing a term in protege
+
+Before you start:
+
+-   make sure you are working on a [here](https://docs.google.com/presentation/d/1M8NZQOIQVswng-so6ROxVeMJfDnzth7BYNj_5MXxEik/edit#slide=id.g9db6baf776_1_0).
+-   make sure you have the editor's file open in Protege (in ODK ontologies, located in: ./src/ontology/ONT-edit.owl) where ONT is the name of your ontology (eg mondo-edit.owl for MONDO)
+
+## Adding, editing, annotating and deleting axioms
+
+#### Adding annotations 
+
+Using Protégé you can add annotations such as labels, definitions, synonyms, database cross references (dbxrefs) to any OWL entity. The panel on the right, named Annotations, is where these annotations are added. OBO Foundry ontologies includes a pre-declared set of annotation properties. The most commonly used annotations are below. 
+
+-   rdfs:label
+-   definition
+-   has_exact_synonym
+-   has_broad_synonym
+-   has_narrow_synonym
+-   has_related synonym
+-   database_cross_reference
+-   rdfs:comment
+
```

Comment:
Mention that, following OBO conventions, a term can have only one each of label, definition and comment (some ontologies have been more relaxed about comment, but this causes problems for releasing OBO versions.)

Reply:
I think Protege won't let you save with more than one label or more than one comment?

Reply:
@nicolevasilevsky, FYI Protégé allows one to save more than one comment to a class (at least in CL). However, that causes CL QC checks to fail, so I ended up having to remove it before merging.

Reply:
ah, thanks for clarifying @bvarner-ebi!